lib/glesv2/glesv2.nit

```diff
@@ -434,12 +434,27 @@ fun glUniform4f(index: Int, x, y, z, w: Float) `{ glUniform4f(index, x, y, z, w)
 
 # Low level array of `Float`
 class GLfloatArray
-	super CArray[Float]
-	redef type NATIVE: NativeGLfloatArray
+	super FinalizableOnce
 
-	redef init(length)
+	var length: Int
+
+	var native_array = new NativeGLfloatArray(length) is lateinit
+
+	fun [](index: Int): Float do return native_array[index]
```

Comment:
usually I would propose to specialize `Sequence[Float]` so you get a lot of free stuff from the interface. unfortunately the current compiler will force expensive float boxing/unboxing because [] and []= are introduced with a non primitive parameter. we should have some heterogeneous optimization for generics.